src/sugar3/graphics/tray.py

```diff
@@ -261,6 +290,10 @@ def __init__(self, **kwargs):
         self._box.pack_start(self._viewport, True, True, 0)
         self._viewport.show()
 
+        self._add_bookmark = _TrayToolButton('list-add')
+        self._box.pack_start(self._add_bookmark, False, False, 0)
+        self._add_bookmark.show()
```

Comment:
We probably shouldn't add this button to every tray - otherwise we need to update every activity that uses a tray to make use of the button.

What activities are using the tray button now?  How can we make an api that let's them add a traytoolbutton only if they want it?